deepspeed/module_inject/replace_module.py

```diff
@@ -386,6 +393,13 @@ def replace_fn(child, _policy, layer_id=0, prefix="", state_dict=None):
                                                container=container_g)
                     sds = [None for _ in sds]
                     gc.collect()
+        embedding_weight = None
```

Comment:
Could this be refactored into a function? It looks like duplicate of other change.

Reply:
done